src/main/java/com/nicoislost/command/Commands.java

```diff
@@ -0,0 +1,71 @@
+package com.nicoislost.command;
+
+import com.mojang.brigadier.CommandDispatcher;
+import com.mojang.brigadier.arguments.BoolArgumentType;
+import com.mojang.brigadier.arguments.IntegerArgumentType;
+import com.nicoislost.Zooms;
+import net.fabricmc.fabric.api.client.command.v2.FabricClientCommandSource;
+import net.minecraft.text.Text;
+import net.minecraft.util.Formatting;
+
+import static com.nicoislost.ZoomOMatic.CONFIG;
+import static com.nicoislost.util.ClientCommandUtil.argument;
+import static com.nicoislost.util.ClientCommandUtil.literal;
+
+/**
+ * A class for registering commands
+ */
+public class Commands {
+    public static void register(CommandDispatcher<FabricClientCommandSource> dispatcher) {
```

Comment:
This seems very interesting, as some trying to get there head around mod development, I'd love some more information about this - ie what it's doing; how it works etc ... just some friendly comments ;)